PYME/LMVis/pipeline.py

```diff
@@ -750,9 +755,10 @@ def OpenFile(self, filename= '', ds = None, **kwargs):
         if 'zm' in dir(self):
             del self.zm
         self.filter = None
-        self.mapping = None
+        # self.mapping = None
```

Comment:
```suggestion
        self.mapping = None
```

We *should* reset the mapping when reloading data